### ISSUE TITLE
spring boot integration test is failing

### ISSUE THREAD

**Issue by habeep** [resilience4j/resilience4j-spring-boot2-demo#60]
Hello,
we are trying to write the integration/unit test but it is failing or not working as we expected. can you please help us?

dependency details:

<java.version>1.8</java.version>

          <dependency>
                <groupId>org.springframework.boot</groupId>
                <artifactId>spring-boot-starter-parent</artifactId>
                <version>2.3.10.RELEASE</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>
        <dependency>
            <groupId>io.github.resilience4j</groupId>
            <artifactId>resilience4j-spring-boot2</artifactId>
            <version>1.7.0</version>
        </dependency>

Basically I tried to run the integration test but it is failing. can you please help me to resolve this issue?

Do you have any sample which help me to refer and correct the issue?
